msg_types.go

```diff
@@ -20,6 +26,16 @@ const (
 	PolicyPlacementMsgKey = "PolicyPlacement"
 	// MinimalPolicyComplianceMsgKey - minimal policy compliance message key.
 	MinimalPolicyComplianceMsgKey = "MinimalPolicyCompliance"
+	// ControlInfoMsgKey - control info message key.
+	ControlInfoMsgKey = "ControlInfo"
```

Comment:
no need for the keys that are not used in LH. 
for these we can just define the string in the HoH side.